src/Faucet.jsx

```diff
@@ -123,8 +123,9 @@ const Faucet = ({ style, className, url, header, address: initial, classPrefix }
       setLoading(true);
       try {
         const response = await getPERLs(address);
+        const amount = (typeof response.amount !== "undefined") ? ` ${response.amount} ` : " ";
         if (response.result === "ok") {
-          alert("Successfully sent 1000 PERLs.");
+          alert(`Successfully sent${amount}PERLs.`);
```

Comment:
Shouldn't there be a space between `sent` and `${}` ?

Reply:
This was intentional in order to avoid double spaces when there's no amount available.
I improved now it by removing an unnecessary space character, but the ${amount} is still needs to be next 'PERLs'